nautobot/data_validation/models.py

```diff
@@ -195,14 +194,14 @@ def clean(self):
             )
 
         allowed_field_types = (
-            models.DecimalField,
             models.FloatField,
             models.IntegerField,
         )
 
         excluded_field_types = (
             models.AutoField,
             models.BigAutoField,
+            models.DecimalField,
         )
```

Comment:
Is it not possible to allow/support validation of DecimalFields, instead of disallowing them?